string name_accel_gyro
int32 deviceID_accel_gyro
float32 accel_x
float32 accel_y
float32 accel_z
float32 gyro_x
float32 gyro_y
float32 gyro_z
string name_mag
int32 deviceID_mag
float32 mag_x
float32 mag_y
float32 mag_z
string name_alt_temp
int32 deviceID_alt_temp
float32 alt
float32 temp


